meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mmevk/lmp.cfg

```diff
@@ -24,6 +24,10 @@ CONFIG_FIT_SIGNATURE_STRICT=y
 CONFIG_SPL_FIT_SIGNATURE=y
 CONFIG_SPL_FIT_SIGNATURE_STRICT=y
 CONFIG_FIT_VERBOSE=y
+CONFIG_SECONDARY_BOOT_RUNTIME_DETECTION=y
+CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_USE_SECTOR=y
+CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x300
```

Comment:
If we use my recommended change to `include/configs/imx8mm_evk.h` then we can drop these here:
```
CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_USE_SECTOR=y
CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x300
```